packages/cli/src/commands/connector/add.ts

```diff
@@ -24,12 +25,19 @@ const add: CommandModule<
         describe:
           'Add all official connectors.\n' +
           "If it's true, the specified package names will be ignored.",
+      })
+      .option('cloud', {
+        describe: 'Add additional connectors for cloud',
+        type: 'boolean',
+        default: false,
+        hidden: true,
       }),
-  handler: async ({ packages: packageNames, path, official }) => {
+  handler: async ({ packages: packageNames, path, official, cloud }) => {
     const instancePath = await inquireInstancePath(path);
 
     if (official) {
```

Comment:
also it'll be great to throw an error if `--cloud` presents without `--official`

Reply:
updated